programs/10-sysvar-address-checking/secure/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+use anchor_lang::prelude::*;
+
+declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
+
+#[program]
+pub mod insecure {
+    use super::*;
+
+    pub fn check_sysvar_address(ctx: Context<CheckSysvarAddress>) -> Result<()> {
+        msg!("Rent Key -> {}", ctx.accounts.rent.key().to_string());
```

Comment:
This should be renamed as "recommended".

Can we add an additional program under the "secure" title that manually does the rent address check.

Reply:
yea sure